src/bluefinClient.ts

```diff
@@ -141,6 +143,10 @@ export class BluefinClient {
 
   private isZkLogin: boolean = false;
 
+  private isSubAccountEnabled: boolean = false;
+
+  private parentAddress: string;
```

Comment:
Let's discuss these. This assumes that a given subaccount can only be a subaccount for 1 address. But that's not necessarily true

We can consider changing that. But I'm also curious to learn if we really need these new private variables